egi2bids/mff2bids.py

```diff
@@ -44,28 +50,71 @@
     "249", "250", "251", "F9", "253", "254", "255", "256",
     "Cz",
 ]
+
+
 # fmt: on
+def _extract_folder(file):
+    """Extracts a compressed folder to its original form"""
+    # Get the path and file extension
+    file = Path(file)
+    ext = file.suffix
```

Comment:
```suggestion
    ext = "".join(file.suffixes)
    _check_value(ext, (".tar", ".tar.gz", ".zip", ".mff"), "extension")
```

more DRY + use `.suffixes` to handle file extensions.